Makefile

```diff
@@ -0,0 +1,2 @@
+types:
+	pytype --keep-going .
```

Comment:
If we only have one entry in the Makefile that we only use in one place (i.e. in `build.yml`) IMO it is clearer to just 'hardcode' `pytype --keep-going` in the `build.yml`. But the decision is up to you!